packages/enrichPrescriptions/src/enrichPrescriptions.ts

```diff
@@ -29,9 +35,13 @@ export async function lambdaHandler(event: EnrichPrescriptionsEvent) {
   const searchsetBundle = event.fhir
   const statusUpdates = event.StatusUpdates?.Payload
 
-  if (statusUpdates) {
+  if (statusUpdates && statusUpdates.isSuccess) {
     logger.info("Applying status updates.")
     applyStatusUpdates(searchsetBundle, statusUpdates)
+  } else if (statusUpdates && !statusUpdates.isSuccess) {
+    logger.info("Call to get status updates was unsuccessful. Applying temporary status updates.")
+    const statusUpdateRequest = event.statusUpdateData!
+    applyTemporaryStatusUpdates(searchsetBundle, statusUpdateRequest)
   } else {
     logger.info("No status updates to apply.")
```

Comment:
We should be saying temporarily unavailable in all cases where we can't apply updates.

Reply:
Done.